cmd/createRuleWaiver.go

```diff
@@ -85,15 +110,30 @@ func NewCreateRuleWaiverCommand() *cobra.Command {
 				fmt.Println(tableRow)
 			}
 		},
+		Args: func(cmd *cobra.Command, args []string) error {
+			// validate 'resource-id', 'resource-type', 'resource-provider' and 'resource-tag' when 'wildcard-mode=false'
+			if !opts.WildcardMode && (containsWildcards(opts.ResourceID) ||
+				containsWildcards(opts.ResourceType) ||
+				containsWildcards(opts.ResourceProvider) ||
+				containsWildcards(opts.ResourceTag)) {
+				return errors.New("'wildcard-mode=false' must only be used for exact matches. No wildcards are allowed")
+			}
+			if !isValidTag(opts.ResourceTag) {
+				return errors.New("'resource-tag' must be of the form 'key:value'")
+			}
+			return nil
+		},
 	}
 
 	cmd.Flags().StringVar(&opts.Name, "name", "", "Waiver name")
 	cmd.Flags().StringVar(&opts.Comment, "comment", "", "Comment describing the rule waiver purpose")
 	cmd.Flags().StringVar(&opts.RuleID, "rule-id", "", "Rule ID (e.g. FG_R00217, <UUID Custom Rule ID>)")
 	cmd.Flags().StringVar(&opts.EnvironmentID, "environment-id", "", "Environment ID")
-	cmd.Flags().StringVar(&opts.ResourceID, "resource-id", "*", "Resource ID")
+	cmd.Flags().StringVar(&opts.ResourceID, "resource-id", "*", "Resource ID (e.g. resource-123, 'resource-*')")
 	cmd.Flags().StringVar(&opts.ResourceType, "resource-type", "*", "Resource Type (e.g. AWS.S3.Bucket, '*')")
 	cmd.Flags().StringVar(&opts.ResourceProvider, "resource-provider", "*", "Resource Provider (e.g. aws.us-east-1, azure, '*')")
+	cmd.Flags().StringVar(&opts.ResourceTag, "resource-tag", "*", "Resource Tag (e.g. 'env:prod', 'fugue:*', '*')")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&opts.ResourceTag, "resource-tag", "*", "Resource tag (e.g. 'env:prod', 'env:*')")
```

Just thinking we can keep the examples simple.